runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -331,6 +331,104 @@ ClientSessionData::processIllegalFinalFieldModificationList(const std::vector<TR
          }
       }
    }
+void
+ClientSessionData::dumpAllBytecodeProfilingData()
+   {
+   // Open the file where the IP data will be dumped to.
+   // The file will be suffixed with the client ID for which the data is dumped.
+   FILE *logFile = NULL;
+   char filename[128];
+   int retVal = snprintf(filename, sizeof(filename), "ipdata.server.%llu", (unsigned long long)getClientUID());
```

Comment:
Would this end up in the same place the AOTCache is stored on disk? If not, does it make sense for this to also be in the same location?